src/web/pages/cves/Details.jsx

```diff
@@ -77,7 +77,7 @@ const CveDetails = ({entity}) => {
               .map(([name, title]) => (
                 <TableRow key={name}>
                   <TableData>{`${title}`}</TableData>
-                  <TableData>{_(entity[name])}</TableData>
+                  <TableData>{_l(entity[name])}</TableData>
```

Comment:
This change is actually not required here. But it is required that the value of `entity[name]`  is included in the locale file. And for extracting the value into the locale file it must be included somewhere in the javascript code as `_('Value of entity[name]')`. In this case this should be ensured by the [ENTITY_TYPES](https://github.com/greenbone/gsa/blob/main/src/gmp/utils/entitytype.ts#L67) variable already.